src/components/v5/common/ActionSidebar/partials/forms/PaymentBuilderForm/utils.ts

```diff
@@ -18,15 +19,13 @@ export const getPaymentBuilderPayload = (
   colony: Colony,
   values: PaymentBuilderFormValues,
   networkInverseFee: string,
-): CreateExpenditurePayload | null => {
+): CreateExpenditurePayload | CreateStakedExpenditurePayload | null => {
   const colonyTokens = colony.tokens?.items.filter(notNull);
   const rootDomain = findDomainByNativeId(Id.RootDomain, colony);
   const createdInDomain =
-    findDomainByNativeId(values.createdIn, colony) || rootDomain;
-
-  if (!createdInDomain) {
-    return null;
-  }
+    values.decisionMethod === 'Permissions'
```

Comment:
Can we use here the following check?
```suggestion
    values.decisionMethod === DecisionMethod.Permissions
```